projects/1.16/assets/tinkers-archery/tinkersarchery/lang/zh_cn.json

```
@@ -0,0 +1,247 @@
+  "modifier.tinkersarchery.autoaim.description": "箭指向最近的实体",
+  "modifier.tinkersarchery.autoaim.flavor": "我从不失手",
+  "modifier.tinkersarchery.blazing": "炽热",
+  "modifier.tinkersarchery.blazing.description": "箭着火了",
+  "modifier.tinkersarchery.blazing.flavor": "我火了！",
+  "modifier.tinkersarchery.bouncing_trajectory": "弹跳轨迹",
+  "modifier.tinkersarchery.bouncing_trajectory.description": "箭在地面上弹跳",
+  "modifier.tinkersarchery.bouncing_trajectory.flavor": "跳一跳",
+  "modifier.tinkersarchery.burst": "迸发",
+  "modifier.tinkersarchery.burst.description": "下次你连续拉弓时，它会拉得更快",
+  "modifier.tinkersarchery.burst.flavor": "每一次击杀都会带来荣誉",
+  "modifier.tinkersarchery.chaining": "连环",
+  "modifier.tinkersarchery.chaining.attack_damage": "连锁伤害",
+  "modifier.tinkersarchery.chaining.description": "杀死敌人后，对下一个敌人造成成倍的伤害",
+  "modifier.tinkersarchery.chaining.flavor": "连杀奖励",
+  "modifier.tinkersarchery.clean": "彻底",
+  "modifier.tinkersarchery.clean.accuracy": "尝尝完全拉伸的精确度",
+  "modifier.tinkersarchery.clean.description": "你的弓越拉得紧，箭就射得越准",
+  "modifier.tinkersarchery.clean.flavor": "简单",
+  "modifier.tinkersarchery.finishing": "终结",
+  "modifier.tinkersarchery.finishing.attack_damage": "半血下的伤害",
+  "modifier.tinkersarchery.finishing.description": "对血量低的敌人造成更多伤害",
+  "modifier.tinkersarchery.finishing.flavor": "朽木凋零 / 寒风刺骨 / 命染黄沙",
+  "modifier.tinkersarchery.gravity_trajectory": "重力轨迹",
+  "modifier.tinkersarchery.gravity_trajectory.description": "箭的动作就像他们应该做的那样，以抛物线的轨迹落在地上",
+  "modifier.tinkersarchery.gravity_trajectory.flavor": "像牛顿和苹果",
+  "modifier.tinkersarchery.groovy": "律动",
+  "modifier.tinkersarchery.groovy.description": "你射得越多，拉弓速度就越快",
+  "modifier.tinkersarchery.groovy.draw_speed": "时髦的拉弓速度",
+  "modifier.tinkersarchery.groovy.flavor": "跟上这节奏",
+  "modifier.tinkersarchery.haste": "匆忙",
+  "modifier.tinkersarchery.haste.1": "匆忙Ⅰ",
+  "modifier.tinkersarchery.haste.2": "匆忙Ⅱ",
+  "modifier.tinkersarchery.haste.3": "匆忙Ⅲ",
+  "modifier.tinkersarchery.haste.4": "匆忙Ⅳ",
+  "modifier.tinkersarchery.haste.5": "匆忙Ⅴ",
+  "modifier.tinkersarchery.haste.description": "神奇的红尘使弓拉得更快!",
+  "modifier.tinkersarchery.haste.flavor": "科学!",
+  "modifier.tinkersarchery.highlander": "远眺",
+  "modifier.tinkersarchery.highlander.1": "远眺Ⅰ",
+  "modifier.tinkersarchery.highlander.2": "远眺Ⅱ",
+  "modifier.tinkersarchery.highlander.3": "远眺Ⅲ",
+  "modifier.tinkersarchery.highlander.4": "远眺Ⅳ",
+  "modifier.tinkersarchery.highlander.5": "远眺Ⅴ",
+  "modifier.tinkersarchery.highlander.description": "箭在高处能射得更远",
+  "modifier.tinkersarchery.highlander.flavor": "直到我的膝盖中了一箭",
+  "modifier.tinkersarchery.launching": "充能",
+  "modifier.tinkersarchery.launching.1": "充能Ⅰ",
+  "modifier.tinkersarchery.launching.2": "充能Ⅱ",
+  "modifier.tinkersarchery.launching.3": "充能Ⅲ",
+  "modifier.tinkersarchery.launching.4": "充能Ⅳ",
+  "modifier.tinkersarchery.launching.5": "充能Ⅴ",
+  "modifier.tinkersarchery.launching.description": "箭射出的速度更快",
+  "modifier.tinkersarchery.launching.flavor": "跑！巴里！跑！",

```

```suggestion
  "modifier.tinkersarchery.launching": "迸射",
  "modifier.tinkersarchery.launching.1": "迸射",
  "modifier.tinkersarchery.launching.2": "迸射+",
  "modifier.tinkersarchery.launching.3": "迸射++",
  "modifier.tinkersarchery.launching.4": "迸射+++",
  "modifier.tinkersarchery.launching.5": "迸射++++",
  "modifier.tinkersarchery.launching.description": "箭矢的出射速度更快",
  "modifier.tinkersarchery.launching.flavor": "给箭矢加速",
```
这个flavor应该老久了（），原文的梗应该是闪电侠
迸射本身有其他意思，不知道这里合不合适